#53 Added OR operator support

partikus opened the pull request:
@defrag @norzechowicz code review please

any advices welcome

norzechowicz commented:
@partikus could you please add some tests of this ``or`` matcher also here: https://github.com/coduo/php-matcher/blob/master/tests/MatcherTest.php ? Just to be sure that everything works fine with Facade

partikus commented:
@norzechowicz I just changed it, I replaced manual build with factory, is it ok?
https://github.com/coduo/php-matcher/pull/74/files#diff-9239b5179e155b531c87cd180c1f4966R19

norzechowicz commented:
@partikus yea, its fine, thanks!
